include/odp/api/spec/init.h

```diff
@@ -341,6 +346,39 @@ int odp_term_local(void);
  */
 int odp_term_global(odp_instance_t instance);
 
+/**
+ * Abnormal ODP termination after a non-recoverable error
+ *
+ * Application may use this function to terminate an ODP instance after facing
+ * a non-recoverable error. Depending on the implementation, this function may
+ * attempt to dump stack and other memory areas, clean up and stop HW
+ * operations and/or perform other actions helpful in postmortem analysis.
+ * Depending on the nature of the error resulting in the abnormal termination,
+ * these actions may partially or completely fail. Flags (ODP_TERM_*) and data
+ * parameters can be used to control and pass additional information to the
+ * termination process.
+ *
+ * Some coordination across threads is required when abnormally terminating, if
+ * other threads continue calling ODP functions during or after termination,
+ * their operation is most likely affected.
+ *
+ * When the function returns, the ODP instance has been destroyed either
+ * completely or partially. Application must not attempt to call any ODP
+ * functions during its remaining lifetime, but terminate as soon as feasible.
+ *
+ * @param instance        Instance handle
+ * @param flags           A bit mask of control flags (ODP_TERM_*), set to 0
+ *                        when no flags
+ * @param data            Additional data, set to 0 when no additional data
+ * @param fmt             Printf-style message to be printed to the ODP log
```

Comment:
Why to pass this? Application can print them self. Right?

Reply:
Carried over from the original panic-function, but true, it might be a bit pointless in this new function, ditched in v2.